packages/core/admin/_internal/cli/index.ts

```diff
@@ -0,0 +1,53 @@
+import type { StrapiCommand } from '@strapi/strapi';
```

Comment:
how did you finally manage to solve the circular dependency? 🤔

Reply:
I don't generate types for the `_internal` package at the moment

Reply:
probably it's what you commented in https://github.com/strapi/strapi/pull/18582#issuecomment-1782643649 . Do you think we could spend 30 min next monday to try that in that branch? That would solve the issue.